docs/documentation/index.html

```diff
@@ -3139,6 +3145,33 @@ <h3>Message Types</h3>
 					</table>
 
 					<div class="row" style="height:40px;"></div>
+
+					<h1>Frequently Asked Questions</h1>
+
+					<div class="panel panel-default">
+						<div class="panel-heading">
+							<h3 class="panel-title">Can I use Sourcetrail on confidental source code?</h3>
+						</div>
+						<div class="panel-body">
+							Yes. Sourcetrail works completely offline and establishes no connection to the internet whatsoever. All data stays on your computer.
+						</div>
+					</div>
+
+					<div class="panel panel-default">
+						<div class="panel-heading">
+							<h3 class="panel-title">What happens to the indexed data?</h3>
+						</div>
+						<div class="panel-body">
+							<p>Sourcetrail does two things with the data collected during indexing:</p>
+							<ol>
+								<li>The data is stored in the <code>.srctrldb</code> file in the same directory as the <code>.srctrlprj</code> file. This database file is used by Sourcetrail to serve its purpose of letting you navigate your source code. And it allows Sourcetrail to reopen the project without reindexing the source code every time.</li>
```

Comment:
You could add that the .srctrldb file is actualy an sqlite database file. The user can inspect it with the [DB Browser for SQLite](https://sqlitebrowser.org/).